app/components/anchor/error_message_component.rb

```diff
@@ -0,0 +1,37 @@
+module Anchor
+  class ErrorMessageComponent < Component
+    def initialize(form_builder:, attribute:, **options)
+      @form_builder = form_builder
+      @attribute = attribute
+      @options = options
+
+      super
+    end
+
+    private
+
+    attr_reader :attribute, :form_builder
+
+    delegate :object, to: :form_builder
+
+    def data
+      { error: "true", testid: }
+    end
+
+    def error_message
+      object.errors.full_messages_for(attribute).to_sentence
+    end
+
+    def options
+      { class: TailwindConstants::ERROR, data: }
+    end
+
+    def render?
+      object.errors.has_key? attribute
+    end
+
+    def testid
+      "error-#{attribute.to_s.parameterize}"
```

Comment:
What about this

```suggestion
      "#{object.model_name.singular}-#{attribute.to_s.parameterize}-error"
```

it would create a test id like `employee-email-error`

Reply:
@jas2250 committed the current format on [BuoySoftware/Infirmary](https://github.com/BuoySoftware/Infirmary/). I'd stick to it unless QA requests a change.

Reply:
We're really only looking for these id's to be pre-pended with "error-".   Could do whatever else you'd like after that.